content/blog/nats-server-29-release.md

```diff
@@ -0,0 +1,489 @@
++++
+date = "2022-09-13"
+draft = true
+title = "NATS Server 2.9 Release"
+author = "Byron Ruth"
+categories = ["General"]
+tags = ["NATS", "Release", "JetStream"]
++++
+
+The NATS core team and contributors are proud to announce the [first release][release] of the 2.9.x series! 🚀
+
+Thank you to the **48(!)** people who contributed to this release through GitHub issues, discussions, and pull requests and to the **hundreds** of people in the Slack community who asked questions, presented use cases, and tested early versions of the 2.9 release. ❤️
```

Comment:
```suggestion
Thank you to the **48(!)** people who contributed to this release through GitHub issues, discussions, and pull requests, and to the **hundreds** of people in the Slack community who asked questions, presented use cases, and tested early versions of the 2.9 release. ❤️
```

Missing comma?

Reply:
Thanks, lots of "and"s so switched to "as well as" to bridge the two points.